src/api/middleware/errorMiddleware.ts

```diff
@@ -12,7 +14,14 @@ class ResponseError extends Error {
   url: string
 
   constructor(clientName: string, response: ConfluenceResponse) {
-    super(`${clientName} - ${response.data().message || `Error, status ${response.status()}`}`)
+    super(
```

Comment:
This change has caused some tests to fail that need to be updated.

The final string now has a trailing space if there are no errors. I would suggest the following instead:

```ts
`Error, status ${response.status()}${response.error() ? ` ${response.error()}` : ''}`
```